apps/web-twig-demo/templates/partials/tabs.html.twig

```diff
@@ -0,0 +1,11 @@
+<ul class="Tabs mb-1100" role="tablist">
```

Comment:
Make the call only once `{% set routeName = app.current_route %}`.

https://symfony.com/doc/current/routing.html#getting-the-route-name-and-parameters

Reply:
> The app.current_route and app.current_route_parameters variables were introduced in Symfony 6.2. Before you had to access _route and _route_params request attributes using app.request.attributes.get().

😁